docs/MAINTAINING.md

```diff
@@ -0,0 +1,43 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Maintaining
+
+- [Maintaining](#maintaining)
+  - [Workflow](#workflow)
+    - [Milestones](#milestones)
+    - [Project Boards](#project-boards)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Maintaining
+
+Current maintainers:
+
+- @code-asher
+- @oxy
+- @jsjoeio
+
+This document is meant to serve current and future maintainers of code-server, but also share openly our workflow for maintaining the project.
+
+## Workflow
+
+The workflow used by code-server maintainers is one that aims to be easy to understood by the community and easy enough for new maintainers to jump in and start contributing on day one.
+
+### Milestones
+
+We operate mainly using [Milestones](https://github.com/cdr/code-server/milestones). This was heavily inspired by our friends over at [vscode](https://github.com/microsoft/vscode).
+
+Here are the Milestones we use and how we use them:
+
+- "Backlog" -> Work not yet planned for a specific release.
+- "On Deck" -> Work under consideration for upcoming milestones.
+- "Backlog Candidates" -> Work that is not yet accepted for the Backlog. We wait for the community to weigh in.
+- "<Month> <Year>" -> Work to be done that month.
+
+### Project Boards
+
+We use project boards sparingly.
+
+Think of this as a place to put miscellaneous things (like testing, clean up stuff, etc). As a maintainer, random todos may come up here and there. This gives you a place to add notes without having to open a new issue (though that may be the better option — use your best judgement).
```

Comment:
IMO if there's work to be done an issue always makes sense. Is the flow for creating a note significantly different than an issue?

Reply:
Main difference is notes aren't counted like Issues are. See creation flow here:
![2021-04-28 10 28 02](https://user-images.githubusercontent.com/3806031/116447205-9488cf80-a80c-11eb-8a3c-43d6015eb756.gif)

I just wanted an option to write a "todo" somewhere without significantly increasing our issue count (which some users use as a metric to indicate a project's healthiness). 

No strong opinion here. Happy to revise if you think everything should be an issue!

Reply:
> given that our release milestones function off of issues, then having it be a dedicated issue instead of a project note may make more sense

Discussed a bit more and going to change this wording to be "This gives you a place to add notes temporarily before opening a new issue"